src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
```

Comment:
use switch for limited amount of condititons

https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md

Reply:
fixed. used switch instead of if